### PR TITLE
docs(API): `scrollend` event fired when scrolling complete in Document and scrollable elements

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds details for the `onscrollend` event which is fired when scrolling is complete inside scrollable elements or a Document.

__Additions:__
- `files/en-us/web/api/document/scrollend_event/index.md` Document `scrollend` event
- `files/en-us/web/api/element/scrollend_event/index.md` element `scrollend` event

__Updates:__
- `files/en-us/web/api/document/scroll_event/index.md` and `files/en-us/web/api/document/scrollend_event/index.md` pages include links to respective `scrollend` pages
- Removed warnings for deprecated [scroll events in UIWebview](https://developer.apple.com/documentation/uikit/uiwebview) as this is deprecated in favor of [wkwebview](https://developer.apple.com/documentation/webkit/wkwebview)

__Related issues and pull requests:__
- [x] BCD https://github.com/mdn/browser-compat-data/pull/18407
- [ ] parent issue https://github.com/mdn/content/issues/22814

### CODE REVIEW

File: files/en-us/web/api/document/scrollend_event/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+Scroll position updates include smooth or instant mouse wheel scrolling, keyboard scrolling, scroll-snap events, or other APIs and gestures which cause the scroll position to update.
+User gestures like touch panning or trackpad scrolling aren't complete until pointers or keys have released.
+If the scroll position did not change, then no scrollend event fires.
+
+For detecting when scrolling inside an element is complete, see {{domxref("Element/scrollend_event", "Element: scrollend event")}}.
+
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}} or set an event handler property.
```
Comment (sideshowbarker) [inline files/en-us/web/api/document/scrollend_event/index.md:21]:
```suggestion
Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
```

File: files/en-us/web/api/document/scrollend_event/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+## Event type
+
+A generic {{domxref("Event")}}.
+
+## Examples
+
+### Using Document `scrollend` with an event listener
+
+The following example shows how to use the `scrollend` with an event listener to detect when the user has stopped scrolling the document.
```
Comment (sideshowbarker) [inline files/en-us/web/api/document/scrollend_event/index.md:37]:
```suggestion
The following example shows how to use the `scrollend` event with an event listener to detect when the user has stopped scrolling the document.
```

File: files/en-us/web/api/document/scrollend_event/index.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+document.addEventListener("scrollend", (event) => {
+  output.innerHTML = `Document scrollend event fired!`;
+});
+```
+
+{{EmbedLiveSample("Using_document_scrollend_with_an_event_listener", "100%", 200)}}
+
+### Using `onscrollend` event handler property
+
+The following example shows how to use `scrollend` event handler property to detect when the user has stopped scrolling the document.
```
Comment (sideshowbarker) [inline files/en-us/web/api/document/scrollend_event/index.md:93]:
```suggestion
The following example shows how to use the `scrollend` event handler property to detect when the user has stopped scrolling the document.
```

File: files/en-us/web/api/element/scrollend_event/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+Scroll position updates include smooth or instant mouse wheel scrolling, keyboard scrolling, scroll-snap events, or other APIs and gestures which cause the scroll position to update.
+User gestures like touch panning or trackpad scrolling aren't complete until pointers or keys have released.
+If the scroll position did not change, then no scrollend event fires.
+
+For detecting when scrolling inside a Document is complete, see the {{domxref("Document/scrollend_event", "Document: scrollend event")}}.
+
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}} or set an event handler property.
```
Comment (sideshowbarker) [inline files/en-us/web/api/element/scrollend_event/index.md:21]:
```suggestion
Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by bsmth** [comment]
Thanks @sideshowbarker 👍🏻

**Pull request merged**